docs/src/AlgebraicGeometry/standard_constructions.md

```diff
@@ -0,0 +1,24 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["standard_constructions.md"]
+```
+
+# Standard Constructions in Algebraic Geometry
+
+This page documents a collection of standard constructions in Algebraic Geometry
+available in Oscar.
+
+## Grassman Plücker Ideal
```

Comment:
Wouldn't it make more sense to have this text in the docstring for `grassman_pluecker_ideal`, so that it is also available in the REPL, and then just include that docstring here?

(Of course right now there are two docstrings for `grassman_pluecker_ideal`, but those could be merged into a single one).

To be clear: I am not saying this needs to be done now; indeed, I'd rather merge this now and improve it later. But then for this later day we might consider this change?

Reply:
sure, I wasn't aware this could be done. That does seem better.

Reply:
I am just talking about the usual way we insert docstrings into the manual, i.e.
````
```@docs
grassman_pluecker_ideal
```
````
Please take a look at other .md files to see how this is done there.